### ISSUE TITLE
use enableImportInjection: true

### ISSUE THREAD

**Issue by felixroos** [getflip/swirl#280]
Would you consider using `enableImportInjection: true` which is available since [stencil v3.2.0](https://github.com/ionic-team/stencil/releases/tag/v3.2.0)? Without it, the dynamic imports generated by stencil break vite production builds (potentially others too).

See https://stenciljs.com/docs/config-extras#enableimportinjection

I ran into this issue while testing a production build, which throws this error in the client:

```
Loading failed for the module with source “/*...*//swirl-app-layout_7.entry.js”.

Uncaught (in promise) TypeError: can't access property "isProxied", l is undefined
```

Without the flag, stencil uses dynamic imports to request the component files at runtime, e.g. import(`./${l}.entry.js`), which cannot be statically analyzed (aka the files are missing), breaking builds that optimize source code based on static analysis (like vite).

**Comment by Sqrrl** [comment]
Hey. Didn't get around to it this week. I'll take a look at it first thing next week.

**Comment by Sqrrl** [comment]
Published `0.65.0-import-injection.0` with the enableImportInjection option enabled. Please let me know if this works.
